redisearch/client_test.go

```diff
@@ -914,15 +914,67 @@ func TestClient_CreateIndex(t *testing.T) {
 
 	// Create docs with a name that has the same phonetic matcher
 	vanillaConnection := c.pool.Get()
-	vanillaConnection.Do("HSET", "create-index-info:doc1", "name", "Jon", "age", 25)
-	vanillaConnection.Do("HSET", "create-index-info:doc2", "name", "John", "age", 20)
+	_, err = vanillaConnection.Do("HSET", "create-index-info:doc1", "name", "Jon", "age", 25)
+	assert.Nil(t, err)
+	_, err = vanillaConnection.Do("HSET", "create-index-info:doc2", "name", "John", "age", 20)
+	assert.Nil(t, err)
 
 	// Wait for all documents to be indexed
-	info, _ := c.Info()
+	info, err := c.Info()
+	assert.Nil(t, err)
+	for info.IsIndexing {
+		time.Sleep(time.Second)
+		info, _ = c.Info()
+	}
+	assert.Equal(t, uint64(2), info.DocCount)
+	assert.Equal(t, false, info.IsIndexing)
+	assert.Equal(t, uint64(0), info.HashIndexingFailures)
+	docs, total, err := c.Search(NewQuery("Jon").
+		SetReturnFields("name"))
+	assert.Nil(t, err)
+	// Verify that the we've received 2 documents ( Jon and John )
+	assert.Equal(t, 2, total)
+	assert.Equal(t, "Jon", docs[0].Properties["name"])
+	assert.Equal(t, "John", docs[1].Properties["name"])
+}
+
+func TestClient_CreateJsonIndex(t *testing.T) {
+	c := createClient("create-json-index")
+	flush(c)
+	version, _ := c.getRediSearchVersion()
+	if version < 20200 {
```

Comment:
Since this variable should be used elsewhere - rather than duplicate, let's set the value in one place and refer to it.